### DISCUSSION TITLE
Question about the different apps named "SagerNet"on F-Droid?

### DISCUSSION THREAD

**Discussion by sudomain** [SagerNet/SagerNet#154] [Q&A] [answered]
Hello,

I want to try using this app but I'm not sure which one I need. There are 6 named "SagerNet" that have been updated in the last month and all of them link to this repo under the `Source Code` section of F-Droid.

EDIT: sorry for not actually asking a question: What are the differences between the 6 apps named "SagerNet"?

**Comment by nekohasekai** [comment, marked as answer]
This is a bug in the F-Droid client, and it displays normally on the web page. By the way, you can distinguish them by icons.

**Reply by sudomain** [reply]
Thank you for the quick answer and sorry for not starting this as a GH discussion.

> This is a bug in the F-Droid client, and it displays normally on the web page

Which webpage? I searched on the Fdroid client app and the Fdroid website, both [show 6 results](https://search.f-droid.org/?q=Sagernet&lang=en)

**Reply by nekohasekai** [reply]
It seems that the search results are also wrong, but it is normal when opened.

**Reply by sudomain** [reply]
Wait are they not just plugins for the main app? E.g. the [XTLS Plugin](https://f-droid.org/en/packages/io.nekohasekai.sagernet.plugin.xtls/)
